packages/web-integration-oc10/appinfo/info.xml

```diff
@@ -15,7 +15,7 @@ For feedback and bug reports, please use the [public issue tracker](https://gith
 	</description>
 	<licence>AGPL</licence>
 	<author>ownCloud</author>
-	<version>2.1.0</version>
+	<version>3.0.0</version>
```

Comment:
Please bump versions in release preparation PRs only. Having this in master before an actual release is misleading.

Reply:
Yea, makes sense, sorry for that.